impeller/renderer/texture_mipmap.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/renderer/texture_mipmap.h"
+#include "impeller/renderer/blit_pass.h"
+#include "impeller/renderer/command_buffer.h"
+
+namespace impeller {
+
+fml::Status AddMipmapGeneration(const std::shared_ptr<Context>& context,
+                                const std::shared_ptr<Texture>& texture) {
```

Comment:
Hmm, this seems simple enough where just doing it inline is easier to parse. At the same time, it's specific enough where I'm not sure we'd want to do exactly this elsewhere.

Reply:
We all agreed that functions should not be longer than one screen, ideally.  You can only get there by breaking out smaller functions.  Already once in the the course of this PR there were 2 places identified where this was supposed to be called.  I can convert it to a function in an anonymous namespace if when this is all over we still are able to accomplish everything we want in InlinePassContext.